### ISSUE TITLE
Request Camera Access and Photo Library Access 

### ISSUE THREAD

**Issue by jonahz5222** [jonahz5222/grocer#12]
This should prompt the first time they try to create a new Receipt

You have to update the info.plist and add code in the view controller

**Comment by jonahz5222** [comment]
What's the status on this? Its done right? Just not merged because it needs the other stuff built first?

**Comment by yuichiu416** [comment]
It's 80% done.
For some reason after selected the photo, it won't do anything.
I think there's some issue when calling the imagepicker controller, I'm
working on it.

Still waiting on the storyboards that needs to access camera/photo library,
which is the edit user screen and the upload receipt screen.
